REDIRECT.en.md

```diff
@@ -0,0 +1,211 @@
+# Guide to URI redirection
+
+This document provides guidance for creating and publishing
+`htaccess` files that allow redirection of semantic resources 
+URIs using the w3id.
+
+## Requirements Notation
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in BCP 14 [RFC2119] [RFC8174] when,
+and only when, they appear in all capitals, as shown here.
+
+## Introduction to redirect
+
+[w3id.org](https://w3id.org/) is a solution from the W3C Permanent Identifier Community Group that allows the addition or modification of permanent identifiers from which to redirect to specific URLs; the process is based on adding one or more folders in the [w3id git repository](https://github.com/perma-id/w3id.org), which MUST contain the `.htaccess` files and `README.md`, possibly organized into subfolders.
+
+In the case of the Catalogue, it is necessary to refer to the [Italy folder of the w3id GIT](https://github.com/perma-id/w3id.org/tree/master/italia), in which the subfolders MUST be added, each for a particular thematic area, which will contain the redirection files, possibly organized in further subfolders, and the `README.md` file. The addition of the `'/Italy'` subfolders will be subject to approval by the Italy Committee; subsequently, they will be managed independently and with direct interfacing between w3id.org and the contacts indicated in the `README.md`. The folder and its subfolders, `.htaccess` files, and `README.md` files MUST be created on the [w3id git repository](https://github.com/perma-id/w3id.org) by the Contributor.
+
+## Publishing htaccess on w3id
+
+### 1. fork of the w3id.org GIT repository
+
+The first step for registering the redirects is the local fork of the
+[w3id GIT Italy folder](https://github.com/perma-id/w3id.org/tree/master/italia). The
+permanent identifiers (URI) will be defined based on the path in which the various `htaccess` files will be inserted.
+In this case, the path to the "root" folder for folder-name must be `"italia/<folder-name>/"`, therefore the namespace of the URIs defined in it will be `w3id.org/italia/<folder-name>/`.
+
+### 2. adding the folder
+
+In the local repository created starting from the fork, it will be necessary to create the `folder-name` folder which will contain the tree of subfolders and the related `.htaccess` files, in addition to the `README.md`.
+
+Below is an example of a folder tree under `"/italia"`:
+
+<pre>
+italia
```

Comment:
Consistently use ``` instead of <pre>

Reply:
Can you please suggest where we should put ``` instead of <pre>